packages/react-core/src/components/Wizard/examples/Wizard.md

```diff
@@ -859,3 +861,141 @@ class WizardInModal extends React.Component {
   }
 }
 ```
+
+### Wizard with drawer
+
+```js isBeta
+import React from 'react';
+import { Button, DrawerActions, DrawerCloseButton, DrawerHead, DrawerPanelContent, Wizard } from '@patternfly/react-core';
+
```

Comment:
Shouldn't we be importing https://www.patternfly.org/v4/utilities/float and https://www.patternfly.org/v4/utilities/spacing here? if the consumer copied and pasted this it would not work unless the imported these. Isn't hat correct @mcoker

Reply:
If they hadn't done so yet, they'd have to import the utilities style sheet, not the classes themselves. i'm not sure we'd want to do that in the example (re-import a whole stylesheet that we already import). But maybe instructions about importing the utilities style sheet can be added as a comment alongside the imports?

Reply:
That's a great idea @nicolethoen ! @tlabaj I combed through our repo, and we already happen to have a lot of demos that use utility classes like this without any kind of documentation. So I think for now, ultimately those changes would be out of scope of the functionality presented in this PR, since we would need to figure out the best way to present those instructions!

 Maybe for all those demos that use utility classes, we can open a separate issue to add those instructions as a comment alongside the imports for all those files? 

Do you think that would be a good game plan so we can get this merged @tlabaj @nicolethoen @mcoker ?

Reply:
So maybe @mcokers comment is correct.  We should not use the utilities in the example.  but if we leave them out, the layout not as shown in the design.
![image](https://user-images.githubusercontent.com/39532947/175645261-d0e37a7e-6f3e-46d2-a0e0-015f28851764.png)

Maybe the example should demo the  button inline in a paragraph variation Then we just demo this variation with the action demo?
Another question I would ask is would we want adding the toggle button and having it aligned correctly work without needing to apply utility classes, or are we ok with it not aligning by default, if that is ok, we can just remove the classes from the example like @mcoker mentioned above.

cc @mcarrano

Reply:
> are we ok with it not aligning by default, if that is ok, we can just remove the classes from the example 

@tlabaj my thinking is that the wizard component doesn't implement the layout/design, and they can do it any number of ways. So I think just presenting the default functionality of the prop in the example is good, and users can look at the demos for design/layout examples.

Reply:
@mcoker I agree.  So @jenny-s51 Can you just remove the utility class from the example.  The demo will show how to lay it out.

Reply:
Good call - updated to remove the utility classes